### DISCUSSION TITLE
Request a andriod version? Moonlight is already for andriod.

### DISCUSSION THREAD

**Discussion by ThetaX55** [LizardByte/community#468] [Sunshine] [answered]
A app for android would be nice.
To stream the android phone to moonlight.
Moonlight is already a app on android.

**Comment by ReenigneArcher** [comment, marked as answer]
This project is not targeting Android and wouldn't be easy to make this work on Android. If anything that should be a separate project altogether. I don't plan on working on that but feel free to if you so desire.

My suggestion though would be to use an Android emulator on a PC.

**Reply by EWEDLCM** [reply]
Yes, I have the same idea！
Because most of today's TVS have good hardware quality but poor software performance. 
so, If we can run Sunshine on Android, then we can install Sunshine in an Android emulator and install Moonlight on an old smart TV or TV box, and we can resurrect the TV. 
According to my test，my SONY TV has trouble running tvbox but it can run moonlight smoothly！！
